lib/SILGen/SILGenBuilder.cpp

```diff
@@ -156,6 +155,7 @@ InitExistentialValueInst *SILGenBuilder::createInitExistentialValue(
 
 InitExistentialMetatypeInst *SILGenBuilder::createInitExistentialMetatype(
     SILLocation loc, SILValue metatype, SILType existentialType,
+    CanType formalConcreteInstanceType,
```

Comment:
Do you really need this? The instance type of a metatype is unlowered, so it should match the formal instance type.

Reply:
It was necessary to support the hack of substituting out opaque types everywhere in SILGen, but I can back it out now.